mri/operators/fourier/non_cartesian.py

```diff
@@ -378,9 +385,127 @@ def adj_op(self, x):
         return img * np.sqrt(np.prod(self.Kd))
 
 
+class gpuNUFFT:
+    """  GPU implementation of N-D non uniform Fast Fourrier Transform class.
+
+    Attributes
+    ----------
+    samples: np.ndarray
+        the mask samples in the Fourier domain.
+    shape: tuple of int
+        shape of the image (necessarly a square/cubic matrix).
```

Comment:
that's a shame, where does this requirement come from?

Reply:
Oh damn, missed `not` :face_with_head_bandage: